templates/guides/import-script.md

```diff
@@ -0,0 +1,115 @@
+
+1. If you are using a Unix-based systems, add execute permission to the script file: 
+```
+chmod +x terraform-provider-astro-import-script_&lt;version-number&gt;_&lt;os&gt;_&lt;arc&gt;
+```
+2. Run the import script. Insert the script's version, your computer's operating system, and your computer's architecture for `<version-number>`, `<os>` and `<arc>`.
+
+- On Unix-based systems:
+```
+./terraform-provider-astro-import-script_&lt;version-number&gt;_&lt;os&gt;_&lt;arc&gt; [options]
+```
+- On Windows:
+
+```
+.\terraform-provider-astro-import-script_&lt;version-number&gt;_&lt;os&gt;_&lt;arc&gt;.exe [options]
+```
+
+### Options
+- `-resources`: Comma-separated list of resources to import. Accepted values are workspace, deployment, cluster, api_token, team, team_roles, user_roles. If not provided, all resources are imported.
+
+-> Ensure you have the necessary permissions in your Astro organization to access the resources you're attempting to import.
+
+- `-token`: API token to authenticate with the Astro platform. If not provided, the script will attempt to use the ASTRO_API_TOKEN environment variable.
+- `-organizationId`: Organization ID to import resources from.
+- `-runTerraformInit`: Run terraform init after generating the import configuration. Used for initializing the Terraform state in our GitHub Actions.
+- `-help`: Display help information.
+
+You following is an example output from running the import script with `-resources deployment` and `-organizationId cf23qgwp001ag01qf0o8er413`:
+```
+Terraform Import Script Starting
+Resources to import:  [deployment]
+Using organization ID: cm23qgwp001ap01qm0o3er493
+Terraform version 1.9.7 is installed and meets the minimum required version.
+Importing deployments for organization cm23qgwp001ap01qm0o3er493
+Importing Deployments: [cf23qgwp001ag01qf0o8er413]
+Successfully handled resource deployment
+Successfully wrote import configuration to import.tf
+generated.tf does not exist, no need to delete
+terraform.tfstate does not exist, no need to delete
+astro_deployment.deployment_cf23qgwp001ag01qf0o8er413: Preparing import... [id=cf23qgwp001ag01qf0o8er413]
+astro_deployment.deployment_cf23qgwp001ag01qf0o8er413: Refreshing state... [id=cf23qgwp001ag01qf0o8er413]
+
+Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
+  + create
+  ~ update in-place
+
+Terraform will perform the following actions:
+
+...
+
+Plan: 1 to import, 6 to add, 1 to change, 0 to destroy.
+
+generated.tf does not exist. Creating new file with deployment information.
+Generated import for astro_deployment.deployment_cf23qgwp001ag01qf0o8er413
+Successfully updated generated.tf with deployment information.
+Import process completed successfully. The 'generated.tf' file now includes all resources, including deployments.
+Import process completed. Summary:
+Resource deployment processed successfully
+```
```

Comment:
can you run the script again and include an example that doesn't modify one of the resources?